spec/features/view_peeps_spec.rb

```diff
@@ -0,0 +1,56 @@
+# As a Maker
```

Comment:
It's great to have this sort of comments to guide your process. However, I'd remove these comments now as part of refactoring and cleaning up application before submitting.